core/accounts/templates/profile-settings.html

```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
+
+{% block title %}
+    Settings | UWA Case
+{% endblock %}
+
+{% block body %}
+    <div class='container'>
+            <p> {{ message }} </p>
+    </div>
+    <div class='container'>
+    	    <div class='container'>
+        <div class='signup-form'>
+            <form method="post">
+                {% csrf_token %}
+                <h2>Settings</h2>
+                <p class="hint-text">Reset your account details.<br>
```

Comment:
No closing \</p>, (unneeded \<br>?)

Does this hint line need to be here.

If it does then it should read something like “modify your account” rather than reset.